Library/Homebrew/formula_installer.rb

```diff
@@ -942,13 +943,25 @@ def post_install
     @show_summary_heading = true
   end
 
-  def pour
+  def prefetch
+    compute_dependencies unless ignore_deps?
+    fetch_bottle if pour_bottle?
```

Comment:
Will need to fetch non-bottles here, too. See `def build` for the `def pour` equivalent for non-bottles that you'll need to hook into.